include/robot_localization/ros_filter_utilities.h

```diff
@@ -97,6 +98,72 @@ void stateToTF(const Eigen::VectorXd &state, tf2::Transform &stateTF);
 //!
 void TFtoState(const tf2::Transform &stateTF, Eigen::VectorXd &state);
 
+
+//! @brief Class to determine if it is time to publish
+class TimeToPublish
+{
+
+public:
+
+  //! @brief Default constructor
+  TimeToPublish()
+    : period_(0.02)
+    , last_(ros::WallTime::now().toSec())
```

Comment:
Why `WallTime`? How would this work on simulation? For example, if we pause gazebo, what would happen?